Framework/CurveFitting/test/Functions/valgrind.h

```diff
@@ -3798,8 +3798,8 @@ typedef struct {
 
 /* These macros are used by tools -- they must be public, but don't
    embed them into other programs. */
-#define VG_USERREQ_TOOL_BASE(a, b) ((unsigned int)(((a)&0xff) << 24 | ((b)&0xff) << 16))
-#define VG_IS_TOOL_USERREQ(a, b, v) (VG_USERREQ_TOOL_BASE(a, b) == ((v)&0xffff0000))
+#define VG_USERREQ_TOOL_BASE(a, b) ((unsigned int)(((a) & 0xff) << 24 | ((b) & 0xff) << 16))
+#define VG_IS_TOOL_USERREQ(a, b, v) (VG_USERREQ_TOOL_BASE(a, b) == ((v) & 0xffff0000))
```

Comment:
This format change might be okay, I just wanted to flag it to check with you?

Reply:
Space near a bitwise operator should be fine